Formula/vile.rb

```diff
@@ -0,0 +1,35 @@
+class Vile < Formula
+  desc "Vi Like Emacs Editor"
+  homepage "https://invisible-island.net/vile/"
+  url "https://invisible-island.net/archives/vile/current/vile-9.8v.tgz"
+  sha256 "240edec7bbf3d9df48b3042754bf9854d9a233d371d50bba236ec0edd708eed5"
+  license "GPL-2.0-or-later"
+
+  depends_on "expect" => :test
+
+  uses_from_macos "flex" => :build
+  uses_from_macos "groff" => :build
+  uses_from_macos "libiconv"
+  uses_from_macos "ncurses"
+  uses_from_macos "perl"
+
+  def install
+    system "./configure", *std_configure_args,
+                          "--disable-imake",
+                          "--enable-colored-menus",
+                          "--with-ncurses",
+                          "--with-screen=ncurses",
+                          "--without-x"
+    system "make", "install"
+  end
+
+  test do
+    require "pty"
+
+    PTY.spawn(bin/"vile") do |_, w,|
+      w.write ":w new\n"
+      w.write ":q\n"
+    end
```

Comment:
Yea, ok, this didn't work. I guess we can use `expect` 🙂

Reply:
I'll change it back to expect - :)

Reply:
https://github.com/Homebrew/homebrew-core/pull/106086/commits/701d9ee62b34b7b60994866fc7fe723025359f47 ...